xml/slurm.xml

```diff
@@ -729,7 +729,7 @@ systemctl enable slurmd.service</screen>
         <xref linkend="st-slurm-configuration-distribute"/>.
        </para>
        <note>
-        <title>Memory Size Seen by <literal>slurmd</literal> Can Change on Update</title>
+        <title>Memory size seen by <literal>slurmd</literal> can change on update</title>
```

Comment:
We only recently switched from Title Case to sentence case for titles and this happens quite often.